### DISCUSSION TITLE
Vercel Edge Function with Sveltekit, trying to use node:events

### DISCUSSION THREAD

**Discussion by alexis-wei** [vercel/community#5897] [Help] [answered]
### Summary

I want one of my sveltekit api routes to run as an edge function instead of serverless (the rest of them are as node). This route makes use of `EventEmitter` in `node:events`,
My build is failing:

```
import { EventEmitter } from "node:events";
                                          ~~~~~~~~~~~~~

  The package "node:events" wasn't found on the file system but is built into node. Are you
  trying to bundle for node? You can use "platform: 'node'" to do that, which will remove
  this error.
 ```

https://vercel.com/docs/functions/runtimes/edge-runtime#compatible-node.js-modules

not sure the question belongs here or on sveltekit side. If anyone has any idea how, would really appreciate it!

### Steps to Reproduce

Using sveltekit with vercel-adaptor

Has an api route `/api/events`
 and 
 `export const config = {
  runtime: 'edge',
};`

**Comment by dummdidumm** [comment, marked as answer]
Since version 5.3 of `@sveltejs/adapter-vercel` this is supported out of the box without any configuration. If you are on SvelteKit 2 you should bump your adapter major version and it should work. If you're on SvelteKit 1 and can't update then you need the external configuration. Inside your svelte.config.js :

```js
import adapter from "@sveltejs/adapter-vercel";

/** @type {import('@sveltejs/kit').Config} */
const config = {
  kit: {
    adapter: adapter({
      external: [
        'async_hooks', 'events', 'buffer', 'assert', 'util',
        'node:async_hooks', 'node:events', 'node:buffer', 'node:assert', 'node:util'],
    }),
  },
};

export default config;
```
